### PR TITLE
Adding product review functionality

### LINKED ISSUE #45: Adding product review functionality
## REQUIREMENTS / DESCRIPTION

As a customer, I want to live a product review in order to provide feedback about the product.

### PR DESCRIPTION
**Pull request by garzo94**
# Description

This pull request adds a new feature that allows customers to leave reviews for products. With this feature, customers can provide feedback on their experience with a particular product, which can be helpful for other potential buyers.

Captura de pantalla (209)

### CODE REVIEW

**Review by Camilapinz18 — APPROVED** [review]

File: src/components/DetailScreen/ReviewThisProduct.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  const [visibleError, setVisibleError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+  const onDismissSnackBar = () => setVisibleSuccess(false);
+  const onDismissSnackBarError = () => setVisibleError(false);
+  const [review, setReview] = useState();
+  const [rating, setRating] = useState();
+
+  const addReview = async () => {
+    try {
+      // Get the token from AsyncStorage
```
Comment (yovanha) [inline src/components/DetailScreen/ReviewThisProduct.jsx]: Please remove your commented lines

File: src/components/ProductDetailCard.jsx [hunk 2]
```diff
@@ -116,7 +119,9 @@ const ProductDetailCard = () => {
           </View>
         </View>
       ) : (
-        <Text>Loading...</Text>
+        <View style={productDetailCard.cardContainer}>
+          <Loading />
+        </View>
       )}
 
       {/* Reviews */}
```
Comment (yovanha) [inline src/components/ProductDetailCard.jsx]: Do you need this line?

File: src/components/DetailScreen/ReviewThisProduct.jsx (hunk 1 above)
Comment (garzo94) [reply src/components/DetailScreen/ReviewThisProduct.jsx]: Removed

**Review by IvanGodinez21 — APPROVED** [review]

**Review by yovanha — APPROVED** [review]
Good job!

**Pull request merged**